From 6bc47c339363e10567c6240def6b54b370c4ef63 Mon Sep 17 00:00:00 2001
From: Jason Rogena <j.rogena@cgiar.org>
Date: Wed, 26 Mar 2014 11:04:52 +0300
Subject: [PATCH 127/213] Version 0.8 Revision 3: Farmer does not have to
 specify extension personel during registration. SMS Server number changed to
 +254708944137

---
 MistroFarmer/src/main/AndroidManifest.xml                           | 4 ++--
 .../org/cgiar/ilri/mistro/farmer/FarmerRegistrationActivity.java    | 6 +++---
 .../src/main/java/org/cgiar/ilri/mistro/farmer/LandingActivity.java | 2 +-
 .../main/java/org/cgiar/ilri/mistro/farmer/backend/DataHandler.java | 2 +-
 4 files changed, 7 insertions(+), 7 deletions(-)

diff --git a/MistroFarmer/src/main/AndroidManifest.xml b/MistroFarmer/src/main/AndroidManifest.xml
index 0bd7f29..1bdd249 100644
--- a/MistroFarmer/src/main/AndroidManifest.xml
+++ b/MistroFarmer/src/main/AndroidManifest.xml
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     package="org.cgiar.ilri.mistro.farmer"
-    android:versionCode="1"
-    android:versionName="1.0" >
+    android:versionCode="2"
+    android:versionName="Version 0.8 Revision 3" >
 
     <uses-sdk
         android:minSdkVersion="7"
diff --git a/MistroFarmer/src/main/java/org/cgiar/ilri/mistro/farmer/FarmerRegistrationActivity.java b/MistroFarmer/src/main/java/org/cgiar/ilri/mistro/farmer/FarmerRegistrationActivity.java
index e5f72ea..371a4d0 100644
--- a/MistroFarmer/src/main/java/org/cgiar/ilri/mistro/farmer/FarmerRegistrationActivity.java
+++ b/MistroFarmer/src/main/java/org/cgiar/ilri/mistro/farmer/FarmerRegistrationActivity.java
@@ -151,7 +151,7 @@ public class FarmerRegistrationActivity extends SherlockActivity implements View
     {
         setTitle(Locale.getStringInLocale("farmer_registration",this));
         fullNameTV.setText(" * "+Locale.getStringInLocale("full_name",this));
-        extensionPersonnelTV.setText(" * "+Locale.getStringInLocale("extension_p",this));
+        extensionPersonnelTV.setText(Locale.getStringInLocale("extension_p",this));
         mobileNumberTV.setText(" * "+Locale.getStringInLocale("mobile_number",this));
         numberOfCowsTV.setText(" * "+Locale.getStringInLocale("number_of_cows",this));
         registerButton.setText(Locale.getStringInLocale("register",this));
@@ -290,10 +290,10 @@ public class FarmerRegistrationActivity extends SherlockActivity implements View
             Toast.makeText(this,mobileNoETEmptyWarning,Toast.LENGTH_LONG).show();
             return false;
         }
-        if(extensionPersonnelET.getText().toString() == null || extensionPersonnelET.getText().toString().length() == 0){
+        /*if(extensionPersonnelET.getText().toString() == null || extensionPersonnelET.getText().toString().length() == 0){
             Toast.makeText(this, Locale.getStringInLocale("enter_extension_personnel_name", this), Toast.LENGTH_LONG).show();
             return  false;
-        }
+        }*/
         if(isInFarm && (longitude == null || longitude.length() == 0 || latitude == null || latitude.length() == 0)) {
             Toast.makeText(this,Locale.getStringInLocale("gps_narrowing_down_on_loc",this),Toast.LENGTH_LONG).show();
             return false;
diff --git a/MistroFarmer/src/main/java/org/cgiar/ilri/mistro/farmer/LandingActivity.java b/MistroFarmer/src/main/java/org/cgiar/ilri/mistro/farmer/LandingActivity.java
index 708a6b2..19d8b01 100644
--- a/MistroFarmer/src/main/java/org/cgiar/ilri/mistro/farmer/LandingActivity.java
+++ b/MistroFarmer/src/main/java/org/cgiar/ilri/mistro/farmer/LandingActivity.java
@@ -77,7 +77,7 @@ public class LandingActivity extends SherlockActivity implements View.OnClickLis
 
         //init text according to locale
         initTextInViews();
-        Toast.makeText(this, "Version 0.8 Revision 2", Toast.LENGTH_LONG).show();
+        Toast.makeText(this, "Version 0.8 Revision 3", Toast.LENGTH_LONG).show();
     }
 
     public boolean onCreateOptionsMenu(Menu menu) {
diff --git a/MistroFarmer/src/main/java/org/cgiar/ilri/mistro/farmer/backend/DataHandler.java b/MistroFarmer/src/main/java/org/cgiar/ilri/mistro/farmer/backend/DataHandler.java
index 66dd412..fa8749f 100644
--- a/MistroFarmer/src/main/java/org/cgiar/ilri/mistro/farmer/backend/DataHandler.java
+++ b/MistroFarmer/src/main/java/org/cgiar/ilri/mistro/farmer/backend/DataHandler.java
@@ -58,7 +58,7 @@ public class DataHandler
     private static final int HTTP_RESPONSE_TIMEOUT =20000;
     //private static final String BASE_URL="http://192.168.2.232/~jason/MistroFarmerProject/web";
     //private static final String BASE_URL="http://10.0.2.2/~jason/MistroFarmerProject/web";
-    public static final String SMS_SERVER_ADDRESS = "+254723572302";
+    public static final String SMS_SERVER_ADDRESS = "+254708944137";
     //private static final String BASE_URL="http://hpc.ilri.cgiar.org/~jrogena/mistro_web";
     private static final String BASE_URL="http://azizi.ilri.cgiar.org/ngombe_planner";
     public static final String FARMER_REGISTRATION_URL="/php/farmer/registration.php";
-- 
2.7.1

